tests/test_layout.py

```diff
@@ -413,6 +415,15 @@ class AsIsLayoutStrategyTest(unittest.TestCase):
     def setUp(self) -> None:
         self.strategy = AsIsLayoutStrategy()
 
+        path_includes_drive: bool = (
+            sys.version_info.major == 3
+            and sys.version_info.minor >= 13
+            and os.name == "nt"
+        )
```

Comment:
Can you refactor this check out to a function, in case we want to re-use it elsewhere in tests? Thanks.

Reply:
Indeed I can! should be able to do that very soon.